service/src/main/java/uk/gov/hmcts/reform/fpl/config/GeneralInboxLookupConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.fpl.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@lombok.Data
+@Service
```

Comment:
Does this need to be a service?

Reply:
its Configuration so change it to `@Configuration'